src/datepicker/bs-daterangepicker.spec.ts

```diff
@@ -0,0 +1,216 @@
+import { Component, ViewChild } from '@angular/core';
+import {
+    BsDaterangepickerDirective,
+    BsDaterangepickerConfig,
+    BsDatepickerModule,
+    BsDaterangepickerContainerComponent
+} from '.';
```

Comment:
please don't import from `.`, please import from specific files

Reply:
Hi, I have changed it in this file, but there are few other spec files which import from `.`. Maybe going forward we need to modify those files as well. Thanks!

Reply:
Yeah, will do that